examples/perl/ws_perl.pl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/perl
+
+#
+# /usr/local/websocketd/ws_perl.pl
+#
+# Once started up, this script keeps running and checking for the existance of files.  If a file exists, it 
+#   will send it to browser.
+#
+
+use strict;
+use Data::Dumper;
+use POSIX;
+
+# Autoflush output
```

Comment:
Maybe "WHY" is more important for people looking at examples, how about "Turn off buffering so output immediately readable by websocketd"